spec/kafka_source_spec.rb

```diff
@@ -308,5 +308,72 @@ def self.kafka_producers
                                             }, widgets[1].id)
       end
     end
+
+    context 'with AR models that implement the kafka_producer interface' do
+      before(:each) do
+        # Dummy class we can include the mixin in. Has a backing table created
+        # earlier and has the import hook disabled
+        class WidgetDeprecated < ActiveRecord::Base
```

Comment:
To avoid polluting constant names, it's best to make this an anonymous class and use `stub_const`:

```
deprecated_class = Class.new(ActiveRecord::Base) do
   include Deimos::KafkaSource
   ...
end
stub_const('WidgetDeprecated', deprecated_class)

Reply:
Ok, just updated!